organizations/admin/classes/api_client/KohaClient.php

```diff
@@ -0,0 +1,135 @@
+<?php
+
+require '../resources/api_client/vendor/autoload.php';
+require 'ILSClient.php';
```

Comment:
I've opened a PR (#299) that will autoload interfaces with the main CORAL autoloader if they're placed under `[module]/admin/interfaces/`

That could remove the need for the requires here down the road.

Reply:
Sure. If PR #299 is merged before this one, I'll update this PR accordingly.

Reply:
Since PR#299 has been merged to development, I moved the ILSClient interface in the interfaces directory.